code/modules/fishing/fishing_spots.dm

```diff
@@ -700,3 +699,21 @@ datum/fishing_spot/golden_toilet
 	/obj/item/fish/firefish = 5,\
 	/obj/item/fish/yellow_tang = 15,\
 	/obj/item/fish/mandarin_fish = 5)
+
+//ice cream machine
+/datum/fishing_spot/ice_cream_machine
+	fishing_atom_type = /obj/submachine/ice_cream_dispenser
+	rod_tier_required = 2
+	fish_available = list(/obj/item/reagent_container/food/snacks/condiment/chocchips = 40,\
+	/obj/item/reagent_container/food/snacks/condiment/custard = 30,\
+	/obj/item/reagent_container/food/snacks/condiment/cream = 20,\
+	/obj/item/reagent_container/food/snacks/condiment/syrup = 15,\
+	/obj/item/raw_material/ice = 35,\
+	/obj/item/reagent_contain/food/snacks/ice_cream/goodrandom)
+
+
+//databank
+/datum/fishing_spot/databank
```

Comment:
this is really stretching the concept of fishing lol, should probably have more than one fish at least

Reply:
I don't mind a few silly ones, but yeah maybe this spot should include some random tapes/computer gear too?